### ISSUE TITLE
junos_rpc task name throws error with latest galaxy

### ISSUE THREAD

**Issue by ejmmanning** [Juniper/ansible-junos-stdlib#363]
I am getting the Fatal: error Duleing Junos RPC Models that are list in the day one book. How do I fix it

**Comment by rsmekala** [comment]
@stacywsmith , This issue is regarding using old module task names after installing latest version of Juniper.junos role(v2.0.2). It works fine on using new module names.

Using older names:
```
  tasks:
    - name: Get interface info
      junos_rpc:
        host={{ inventory_hostname }}
        rpc=get-interface-information
```
The above code produces the following error message:
```
TASK [Get interface info] ****************************************************************************
fatal: [junos]: FAILED! => {"changed": false, "failed": true, "msg": "Unsupported parameters for (junos_rpc) module: _module_name,_module_utils_path,format,passwd,user Supported parameters include: args,host,output,password,port,provider,rpc,ssh_keyfile,timeout,transport,username"}

```

Using newer module names:
```
  tasks:
    - name: Get interface info
      juniper_junos_rpc:
        host={{ inventory_hostname }}
        rpc=get-interface-information
```
Result is OK.
```
TASK [Get interface info] ****************************************************************************
ok: [junos]
```

To support the usage of older names, you had created symlinks to the respective files, but the symlink files start with an "_" , and the above error occurs. Was there is any specific reason to have names start with an  `_` (e.g _junos_rpc.py)?
If I remove the `_`, error is resolved.  Could we remove the "_" from file names?

More details about the issue raised by user can be found here
https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/junos-python-ez/Zo73OPNuHVg/YzCo_5wDBAAJ

**Comment by stacywsmith** [comment]
The names which begin with `_` are Ansible's method of indicating that a module is deprecated. The symlinks are Ansible's method of having the same module with two different names, but the same code.

You should NOT remove the `_`.

This problem is irrespective of the version of the `Juniper.junos` role and is really an Ansible issue. The `Juniper.junos` role existed before any Junos support in Ansible core, and included a module named `junos_rpc`. Beginning in Ansible 2.1, Ansible introduced core modules and gave them names which started with `junos_`. This was confusing, but not terrible because none of the names directly conflicted with the module names in the `Juniper.junos` role. Then in Ansible 2.3, Ansible included a module in core named `junos_rpc` even though there was a direct naming conflict with the `junos_rpc` module that already existed in the `Juniper.junos` role. 

The confusion created by Ansible's use of module names which also begin with `junos_` was the motivation for renaming the modules in the `Juniper.junos` role to begin with `juniper_junos_`. The module aliases (symlink + name beginning with `_`) was a way of maintaining backwards compatibility why still allowing users to *MOSTLY* use the old names.

The exception is the `junos_rpc` name because there is a direct naming conflict between the `Juniper.junos` module with that name, and the core module with the same name.

The current situation allows you to execute the *core* module by using `junos_rpc` in the task. If you want to use the `Juniper.junos` module, then you must use the new `juniper_junos_rpc` name.

If you remove the `_` from the `_junos_rpc` symlink in this role, then you will allow users to invoke the `Juniper.junos` module with the name `junos_rpc`, but you will have broken the ability for users to invoke the *core* `junos_rpc` module.

The bottom line if you are using Ansible >= 2.3 is:
Use `junos_rpc` in the task if you want the core module.
Use `juniper_junos_rpc` in the task if you want the `Juniper.junos` module.

**Comment by vnitinv** [comment]
@ejmmanning please follow the changes as suggested by @stacywsmith. Closing the issue. Do open if you still face the issue.
